src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,23 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```

Reply:
I tried using lazy, but one of the tests fails (saveWithActorAndCountry_ok) 
thanks for the q&n btw, i have managed to figure it out finally